oshi-core/src/main/java/oshi/software/common/AbstractOSProcess.java

```diff
@@ -72,6 +74,10 @@ public double getProcessCpuLoadBetweenTicks(OSProcess priorSnapshot) {
         return getProcessCpuLoadCumulative();
     }
 
+    @Override
+    public List<OSThread> getThreadDetails() {
+        return null;
```

Comment:
Better to return `Collections.emptyList()` here. I try to be very deliberate with my nulls!

Reply:
Sure.